epub33/core/index.html

```diff
@@ -1569,7 +1569,7 @@ <h3>XML Conformance</h3>
 			</section>
 		</section>
 		<section id="sec-ocf">
-			<h2>Open Container Format</h2>
+			<h2>Open container format</h2>
```

Comment:
I would keep this in caps even after we change the terminology. It's an acronym. We should also add "(OCF)" as the next heading uses that initialism and there's no place to explain it.

```suggestion
			<h2>Open Container Format (OCF)</h2>
```

Reply:
I fixed this in #2250, so you can ignore or commit, but either way it will get updated.